sound/soc/sof/Kconfig

```diff
@@ -55,6 +55,15 @@ config SND_SOC_SOF_DEBUG_PROBES
 	  Say Y if you want to enable probes.
 	  If unsure, select "N".
 
+config SND_SOC_SOF_PCM_DISABLE_PAUSE
+	bool "SOF disable pause push/release"
+	help
+	  This option disables ALSA pause push/release capabilities for
+	  SOF drivers. These capabilities are not used by typical
+	  sound servers such as PulseAudio, PipeWire and CRAS.
+	  Say Y if you want to disable pause push/release
+	  If unsure, select "N".
```

Comment:
I'm assuming this is Y by default ? If not, we should make this a stronger Y in the text i.e. only select N if you are not running one of the above sound servers and need pause.

Reply:
Our CI tests pause_push/pause_release and I'd like to keep it this way.
I'd prefer to keep this opt-in.

Maybe we can reword the text as 

````
 This option disables ALSA pause push/release capabilities for
  SOF drivers. 
 Users or distributions relying on sound servers such as PulseAudio, PipeWire and CRAS should enable 
  this option, since it has no functional impact. 
  Others who rely on ALSA-level tests and/or wish to keep support for pause_push/pause_release should keep this option disabled.
  If unsure, select "N"
````

Reply:
Sorry for the late comment again, but looking at this, this seems very hard for the integrators.. why such an option? For rewinds, the link to power savings is more understandable, but pause/release is just weird. Would be easier we'd just silently disable pause for affected Intel platforms and have some HDA specific option to enable pause (with a fat warning that enabling this option might cause increased power usage). Then we could add that option to sof default kconfig and be able to continue to run the sof-test cases for pause push/release, and rest of the world would get optimal power consumption out-of-the-box.

Reply:
This is one of those Brexit or endianness questions....
My approach was: 
when you use make olddefconfig, you get the *same* behavior as before. I am in favor of opt-in options when the behavior changes, something I learned the hard way when I broke audio on Linus' laptop when the HDaudio legacy driver was no longer selected and the other driver didn't do so well.

I don't mind changing this if ALSA maintainers view this as confusing, so we can try and revert to default if there is consensus.

Reply:
@plbossart Ok, fair enough -- it's true just dropping pause would change behaviour.